include/stout/flags/flags.cc

```diff
@@ -715,20 +696,38 @@ void Parser::SetFieldMessageOrAggregateErrors(
     std::string error;
   } error_collector;
 
-  google::protobuf::TextFormat::Parser text_format_parser;
-  text_format_parser.RecordErrorsTo(&error_collector);
-  if (!text_format_parser.ParseFieldValueFromString(
-          value,
-          field,
-          message)) {
-    errors.insert(
-        "Failed to parse flag '" + name
-        + "' from normalized value '" + value
-        + "' due to protobuf text-format parser error(s): "
-        + error_collector.error);
+  // Parse the value using an overloaded parser if provided.
+  if (overload_parsing_.count(field->message_type()) > 0) {
+    std::optional<std::string> error =
+        overload_parsing_[field->message_type()](
```

Comment:
Rather than calling `map.count(key)` followed by `map[key` and thus doing two identical map lookups, how about using `map.find` above and just dereferencing the non-end iterator here when the lookup succeeds?

Reply:
Skipping this for now as this part of the code was copy/paste and the `count()` pattern is quite prevalent in this codebase.